plugins/modules/cloudflare_dns.py

```diff
@@ -824,6 +869,7 @@ def main():
             ('state', 'absent', ['record']),
             ('type', 'SRV', ['proto', 'service']),
             ('type', 'TLSA', ['proto', 'port']),
+            ('type', 'CAA', ['caa_tag']),
```

Comment:
If `caa_tag` has a default, it does not need to be required.

Reply:
you are right. fixed